indexer/indexer_tests/indexer_tests.pro

```diff
@@ -4,8 +4,9 @@ CONFIG -= app_bundle
 TEMPLATE = app
 
 ROOT_DIR = ../..
-DEPENDENCIES = indexer platform editor geometry coding base protobuf tomcrypt \
-               succinct opening_hours pugixml stats_client \
+DEPENDENCIES = base indexer platform editor geometry coding map search storage \
```

Comment:
Not sure that static linking will be happy with this. Please, reoder. Looks the the order should be like:
```generator

map
search
routing
indexer
storage
platform
base
3party stuff
```